src/dotnet/CarbonAware.WebApi/Program.cs

```diff
@@ -13,11 +13,19 @@
 builder.Services.Configure<CarbonAwareVariablesConfiguration>(builder.Configuration.GetSection(CarbonAwareVariablesConfiguration.Key));
 builder.Services.AddCarbonAwareEmissionServices(builder.Configuration);
 builder.Services.AddCarbonAwareSciScoreServices(builder.Configuration);
+CarbonAwareVariablesConfiguration config = new CarbonAwareVariablesConfiguration();
+
+builder.Configuration.GetSection(CarbonAwareVariablesConfiguration.Key).Bind(config);
 
 builder.Services.AddHealthChecks();
 
 var app = builder.Build();
 
+if (!string.IsNullOrWhiteSpace(config.WebApiRoutePrefix))
```

Comment:
Feels like a potential attack vector here to use the prefix without checking anything about it. What do you think?

Reply:
Good call on security.  This isn't a security concern but writing the reasons here would take a long time.  I'll give you a call and we can talk through it.